bedrock/settings/static_media.py

```diff
@@ -826,6 +807,12 @@
         ),
         'output_filename': 'css/tabzilla-min.css',
     },
+    'tabzilla-static': {
```

Comment:
Doesn't look like this is used anymore after the change to the 2011 annual report?